src/ComponentInstaller.php

```diff
@@ -286,6 +286,12 @@ private function loadModuleClassesDependencies(PackageInterface $package): array
         $installer    = $this->composer->getInstallationManager();
         $packagePath  = $installer->getInstallPath($package);
 
+        /** @psalm-suppress TypeDoesNotContainNull Package path become nullable in composer 2.5.6 */
```

Comment:
Would a `composer.lock` update suffice here, or are we stuck on an older release?

Reply:
Note: if we can use a newer release, then this requires no test, as it's just a type error, and we can document `$this->mapAutoloaders()` to only accept `non-empty-string`